DEVELOPMENT.md

```diff
@@ -46,14 +86,16 @@ If you need to bump the major version, mark it as breaking change in the body of
 ```text
 fix: upgrade dependency X
 
-BREAKING CHANGE: requires minimum Node.js 14 to run
+BREAKING CHANGE: requires minimum Node.js 16 to run
 ```
 
-3. New versions of this action will be released automatically by the CI, see [.github/workflows/main.yml](.github/workflows/main.yml). This will create a new [GitHub release](https://github.com/cypress-io/github-action/releases) and will update the current `/v5` branch. Thus specifying `- uses: cypress-io/github-action@v5` selects the new version automatically.
+3. New versions of this action will be released automatically by the CI, see [.github/workflows/main.yml](.github/workflows/main.yml). This will create a new [GitHub release](https://github.com/cypress-io/github-action/releases) and will update the current `v5` branch. Thus specifying `uses: cypress-io/github-action@v5` selects the new version automatically.
+
+4. The action's CI is configured to use the [default Angular release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js), which means that only `feat:`, `fix:` and `perf:` trigger a release and log this commit to the [releases](https://github.com/cypress-io/github-action/releases) page. Other Angular commit types listed on [Contributing to Angular](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format) can be used for documentation purposes, however they are ignored by the currently configured release process.
```

Comment:
```suggestion
4. The action's CI is configured to use the [default Angular release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js), which means that only `feat:`, `fix:` and `perf:` trigger a release and logs this commit to the [releases](https://github.com/cypress-io/github-action/releases) page. Other Angular commit types listed on [Contributing to Angular](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format) can be used for documentation purposes, however they are ignored by the currently configured release process.
```

Reply:
@emilyrohrbough 

I thought that `log` was correct because

`feat:`, `fix:` and `perf:` (= they) ... log this commit
"they logs" wouldn't be right

I will reword to avoid this construction so the grammar is a bit different and doesn't cause an issue.